yarn.lock

```diff
@@ -7213,7 +7223,7 @@ uint8arrays@^4.0.6:
   dependencies:
     multiformats "^12.0.1"
 
-uint8arrays@^5.0.0, uint8arrays@^5.0.1:
+uint8arrays@^5.0.0, uint8arrays@^5.0.1, uint8arrays@^5.1.0:
```

Comment:
hm, I think uint8arrays existed as a dependency of a dependency but not directly, yet it was being used.